Sources/StreamChatUI/ChatChannelList/ChatChannelListVC_Tests.swift

```diff
@@ -46,50 +86,104 @@ class ChatChannelListVC_Tests: XCTestCase {
         
         NSTimeZone.default = TimeZone(secondsFromGMT: 0)!
     }
-    
-    func test_chatChannelList_isPopulated() {
-        let channel = ChatChannel.mock(
-            cid: .unique,
-            name: "Channel 1",
-            imageURL: TestImages.yoda.url,
-            lastMessageAt: .init(timeIntervalSince1970: 1_611_951_526_000)
+
+    func test_emptyAppearance() {
+        mockedChannelListController.simulate(
+            channels: [],
+            changes: []
         )
```

Comment:
`simulate` is meant to be used to simulate changes. It seems to me what you're looking for here is `simulateInitial`. The behavior is different, and if used properly, it better simulates the real-world behavior.